projects/core/components/primitive-textfield/value-decoration/value-decoration.component.ts

```diff
@@ -50,13 +50,18 @@ export class TuiValueDecorationComponent {
     }
 
     get value(): string {
-        return this.textfield.value;
+        return this.textfield.value || '';
```

Comment:
When is this fallback required?

Reply:
unfortunately, one of the unit tests fails, do you think the problem is in the test?

Reply:
fixed